### DISCUSSION TITLE
there is any js lib?

### DISCUSSION THREAD

**Discussion by brunubarbosa** [dr5hn/countries-states-cities-database#176] [Q&A] [answered]
Hey guys, I'd like to know some js library from this database, do you guys recommend a good one? I'm looking for it but without success

**Comment by dr5hn** [comment, marked as answer]
> Hey guys, I'd like to know some js library from this database, do you guys recommend a good one? I'm looking for it but without success

Hi

Can you check this one ?

https://github.com/harpreetkhalsagtbit/country-state-city

Let me know if this fills up your purpose.

**Reply by dr5hn** [reply]
@brunubarbosa

**Reply by dr5hn** [reply]
There's one more JS lib created by @ovictormacedo
https://github.com/ovictormacedo/country-state-city.js
